pkg/kubectl/cmd/apply.go

```diff
@@ -336,6 +336,12 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out, errOut io.Writer, opti
 			} else {
 				visitedUids.Insert(string(uid))
 			}
+
+			if string(patchBytes) == "{}" {
+				count++
+				cmdutil.PrintSuccess(mapper, shortOutput, out, info.Mapping.Resource, info.Name, false, "nothing to change")
```

Comment:
The output message is of the format `resource "resource name" operation`. E.g. `pod "pause" configured`. "nothing to change" doesn't really fit (`pod "pause" nothing to change`). It should be a past-tense verb. Maybe `unchanged`? Or `does not require changes`?